File: GRUPO 4/ListaSimplePython/node.py
#/*************************
# * UNIVERSIDAD DE LAS FUERZAS ARMADAS ESPE
# * ALUMNO:  Steven Egas,Pablo Dominguez, Esteban Santos
# * FECHA CREACION: Domingo,  10 de noviembre de 2024
# *  FECHA MODIFICACION: Domingo,  12 de noviembre de 2024
# * Nivel: TERCERO     NRC: 1992
# *************************/

from Fecha import Fecha

class Node:
    def __init__(self, nombre1, nombre2, apellido, correo, cedula,fecha):
        self.nombre1 = nombre1
        self.nombre2 = nombre2
        self.apellido = apellido
        self.correo = correo
        self.cedula = cedula
        self.Fecha=fecha
        self.siguiente = None
        self.anterior = None

File: GRUPO 1/SplayTree/node.cpp
/*********
 * UNIVERSIDAD DE LAS FUERZAS ARMADAS ESPE
 * ALUMNOS: Xavier Altamirano, Martin Cordero, Isaac Oña
 * FECHA CREACION: Viernes, 31 de Enero de 2025
 * FECHA MODIFICACION: Lunes, 03 de Febrero de 2025
 * Enunciado del problema: Splaytree
 * Nivel: TERCERO     NRC: 1992
 *********/
#include <iostream>
#include <iomanip>
using namespace std;

class Node {
public:
    int key;
    Node* left;
    Node* right;

    Node(int value) : key(value), left(nullptr), right(nullptr) {}
};